### DISCUSSION TITLE
SystemCheckError: System check identified some issues

### DISCUSSION THREAD

**Discussion by nsabatelli** [modoboa/modoboa#2996] [Help] [answered]
HI,
I updated the modoboa version to version 2.1.2 and the cron jobs update_statistics, cleanlogs, check_mx and communicate_with_public_api all give the same error, which I report below

SystemCheckError: System check identified some issues:

ERRORS:
modoboa.Record.header_from: (fields.E304) Reverse accessor for 'modoboa.Record.header_from' clashes with reverse accessor for 'modoboa_dmarc.Record.header_from'.
HINT: Add or change a related_name argument to the definition for 'modoboa.Record.header_from' or 'modoboa_dmarc.Record.header_from'.
modoboa_dmarc.Record.header_from: (fields.E304) Reverse accessor for 'modoboa_dmarc.Record.header_from' clashes with reverse accessor for 'modoboa.Record.header_from'.
HINT: Add or change a related_name argument to the definition for 'modoboa_dmarc.Record.header_from' or 'modoboa.Record.header_from'.

How can I solve the issue?

Thank you.
Best regards
Nicola

**Comment by tonioo** [comment, marked as answer]
Have you followed upgrade instructions for 2.1.0 ?
https://modoboa.readthedocs.io/en/latest/upgrade.html#id2

**Comment by nsabatelli** [comment]
Hi Antoine,
thank you for the information.

Following the instructions solved the issue.

Best regards
Nicola

Il 05/05/2023 09:09, Antoine Nguyen ha scritto:
>
> Have you followed upgrade instructions for 2.1.0 ?
> https://modoboa.readthedocs.io/en/latest/upgrade.html#id2
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/modoboa/modoboa/issues/2995#issuecomment-1535819058>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AB6B76A473YC5QTFFBDO3TLXESRUHANCNFSM6AAAAAAXWXENFA>.
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>
>

-- 
Ing. Sabatelli Nicola
Divisione Informatica e Telematica
Servizio RUPAR, LargaBanda e Reti
Ufficio Ingegneria di Rete e Sicurezza
Centro Tecnico R.U.P.A.R.
InnovaPuglia S.p.A. 	Strada Provinciale per Casamassima Km. 3
70010 Valenzano (Bari)
Email: ***@***.***

Web: www.innova.puglia.it <http://www.innova.puglia.it> 	Fax: 
+39.080.4670327
Phone: +39.080.4670662
Mobile: +39.335.7148485

Le informazioni trasmesse sono da intendere solo per la persona e/o ente 
a cui sono indirizzate, possono contenere documenti confidenziali e/o 
materiale riservato. Qualsiasi modifica, inoltro, diffusione o altro 
utilizzo relativo alle informazioni trasmesse da parte di persone e/o 
enti diversi dai destinatari indicati è proibito ai sensi della legge 
675/96. Se lei ha ricevuto questa mail per errore per favore contatti il 
mittente e cancelli queste informazioni da ogni computer. Grazie per la 
collaborazione.

This e-mail is confidential and may well also be legally privileged. If 
you have received it in error, you are on notice of its status. Please 
notify us immediately by reply e-mail and then delete this message from 
your system. Please do not copy it or use it for any purpose, or 
disclose its contents to any other person: to do so could be a breach of 
confidence. Thank you for your co-operation.
